pythonfmu/pythonfmu-export/CMakeLists.txt

```diff
@@ -33,7 +33,37 @@ set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 # Dependencies
 # ==============================================================================
 
-find_package(JNI REQUIRED)
+# Force to use stable Python ABI https://docs.python.org/3/c-api/stable.html
+add_compile_definitions(Py_LIMITED_API)
 find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+if(WIN32)
+    set(Python3_LIBRARIES ${Python3_LIBRARY_DIRS}/python3.lib)
+endif()
+message("${Python3_LIBRARIES} ${Python3_LIBRARY_DIRS}")
+
+# ==============================================================================
+# Sources
+# ==============================================================================
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/headers)
+
+set(sources
+        cpp/cppfmu_cs.cpp
+        cpp/fmi_functions.cpp
+        cpp/PyObjectWrapper.cpp
+        cpp/SlaveInstance.cpp
+        )
+
+add_library(pythonfmu-export ${sources})
+target_compile_features(pythonfmu-export PUBLIC "cxx_std_17")
+target_include_directories(pythonfmu-export
+        PUBLIC
+        "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>"
+        PRIVATE
+        "${CMAKE_CURRENT_SOURCE_DIR}"
+        )
 
-add_subdirectory(src/main)
+target_link_libraries(pythonfmu-export
+        PRIVATE
+        Python3::Python
```

Comment:
This is the issue. We should set it manually using  "${Python3_INCLUDE_DIRS}" and ${Python3_LIBRARIES}

Reply:
@fcollonval